Formula/ncurses.rb

```diff
@@ -23,14 +23,24 @@ class Ncurses < Formula
   end
 
   def install
-    system "./configure", "--prefix=#{prefix}",
-                          "--enable-pc-files",
-                          "--with-pkg-config-libdir=#{lib}/pkgconfig",
-                          "--enable-sigwinch",
-                          "--enable-symlinks",
-                          "--enable-widec",
-                          "--with-shared",
-                          "--with-gpm=no"
+    args = %W[
+      --prefix=#{prefix}
+      --enable-pc-files
+      --with-pkg-config-libdir=#{lib}/pkgconfig
+      --enable-sigwinch
+      --enable-symlinks
+      --enable-widec
+      --with-shared
+      --with-gpm=no
+    ]
+
+    on_linux do
+      # Homebrew GCC does not support Ada
+      # https://github.com/Homebrew/linuxbrew-core/pull/2841
+      args << "--without-ada"
+    end
```

Comment:
I want to try to preserve `git blame` here. Can you just put
```ruby
args = []
on_linux do
  # Homebrew GCC does not support Ada
  # https://github.com/Homebrew/linuxbrew-core/pull/2841
  args << "--without-ada"
end
```
at the start of the formula and then add `*args` as the last argument in the list, so you don't need to refactor them all into an array?